src/tools/gyp/pylib/gyp/generator/ninja.py

```diff
@@ -2116,7 +2116,8 @@ def GenerateOutputForConfig(target_list, target_dicts, data, params,
     master_ninja.variable('cc', cc)
     cxx = GetEnvironFallback(['CXX_target', 'CXX'], cxx)
     master_ninja.variable('cxx', cxx)
-    ld = GetEnvironFallback(['LD_target', 'LD'], ld)
+    # Use default value for LD
+    # ld = GetEnvironFallback(['LD_target', 'LD'], ld)
```

Comment:
Could this be avoided? Why patching generic tools? Can't this be achieved by a config e.g. if reading this var is an issue clear the var instead

Reply:
will try for some other options

Reply:
Looks good to me now. Closing this.